src/scenes/home/signup/signup.js

```diff
@@ -15,6 +15,10 @@ import config from 'config/environment';
 import styles from './signup.css';
 
 class SignUp extends Component {
+  static contextTypes = {
```

Comment:
What is this for?  We should stay away from the Context API.  I am working on bringing Redux to the application.